src/components/dialogs/parameters/common/wrapper-input.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useStyles } from '../parameters';
+import * as yup from 'yup';
+import { FormProvider, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
+import React, { useEffect } from 'react';
+import { Box, Grid, Typography } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+const WrapperInput = ({ value, label, callback, validator, children }) => {
```

Comment:
why do we need a separate system based on RHF ? Can we have the code look like other forms ?

Reply:
To discuss later..